src/extension.ts

```diff
@@ -0,0 +1,151 @@
+import * as os from 'os';
+import { workspace, ExtensionContext, window, commands } from 'vscode';
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind
+} from 'vscode-languageclient/node';
+import { promisify } from 'util';
+import { execFile } from 'child_process';
+
+let client: LanguageClient;
+
+const outputChannel = window.createOutputChannel('Wasp Language Server');
+
+export async function activate(context: ExtensionContext) {
+  console.log('..Extension "vscode-wasp" is now active..');
+
+  const config = workspace.getConfiguration('wasp');
+
+  // Compute path to `waspls` executable, based on settings
+  const userDefinedExecutablePath = config.server.executable;
+  let executablePath = userDefinedExecutablePath === '' ? 'waspls' : userDefinedExecutablePath;
```

Comment:
Is this safer than
```
let executablePath = userDefinedExecutablePath || 'waspls'
```
Maybe this makes it more robust?

Reply:
It's probably the same in this case. I've switched it to your suggestion, since that's usually more common in JS code I think.

Reply:
Good! I think it is useful because it also covers any stuff like undefined, null and similar. Sometimes that is good, sometimes not, but here I think it is good.